### ISSUE TITLE
Judges Stories

### ISSUE THREAD

**Issue by Morningstar88** [Morningstar88/Morningstar-Abraxas-Collections#29]
I Was small claims court judge for several years. About half of the parties who appeared in my court were successful. I ruled in favor of:

tenants recovering unrefunded security deposits, and landlords making deductions for damages and unpaid rent;

unsatisfied customers at car repair shops, and unpaid car repair shops stiffed by customers;

Brides unhappy with wedding vendors, and wedding vendors stiffed by unhappy brides;

Dental patients, and dentists;

Family members who loaned money to family members, family members who borrowed money from family members.

I think you get the picture. In every case, I tried to be fair and to listen to all of the facts. Very few cases involved a difficult legal issue, so the facts and the presentation of the facts is crucial. On many days I would have 30 or 40 cases on a morning calendar, allowing for an average of about 6 minutes per case. Focused brevity is a plus.

I would help those who appeared to be confused, nervous or overwhelmed while attempting to keep them on track in developing the facts of the case. On the other hand, I expected my direct questions to be candidly answered. Evasions, feigned misunderstanding, or less than candid answers would almost always doom that party’s case (there was a reason I asked the question).

Often defendants agreed they either owed the money or owed some money just not as much as the plaintiff was demanding. In those cases, I often was able to either arrange for an agreement for installment payments, a compromise settlement, or could bring the plaintiff’s demands down to earth. In those cases, both parties considered the matter successfully resolved.

In summary, the best way to be successful is to have facts supporting your case and an organized and succinct way to present those facts to the small claims court judge.

My service as a small claims court judge affected the way that later I presented evidence in courtrooms as an advocate for my clients.

**Comment by Morningstar88** [comment]
Yes.

A builder took me to small claims court.

I had used him to build an extension to my house. Just after the extension was completed, water stains started to appear on the ceiling (expensive birch-faced ply).

I had kept back the final payment for the building work as I wasn't happy with some of the work done and wanted to wait until all 'snags' were sorted. Phoned builder to ask him to take a look at the roof. Wouldn't pick up phone. Wrote to him; he didn't contact me. Wrote again, describing the problems. No response.

After 6 weeks, I got a roofing firm in. The roof hadn't been laid properly. They relaid it and wrote a report detailing the issues.

Builder contacted me, demanding his money. I refused to pay, saying he hadn't remedied the roofing problems, resulting in my needing to pay someone else to fix it.

He took me to small claims court.

In the court I presented photographs of the damage, the report from the roofing firm.

The judge dismissed the builder's claim, saying that I'd given him ample time to respond and remedy the issues. Since he hadn't responded, I was within my rights to withhold the money. He awarded costs against the builder.

**Comment by Morningstar88** [comment]
I’m not sure what you want to know, but if you feel you are in the right, I’d recommend it as a possible route towards a resolution.

There’s nothing to be scared of, and everyone at court understands that you (and your opponent) are inevitable going to be nervous, so allowances are made for this, and people (including the judges, in my experience) will try to help you out or advise you on legal niceties.

What other advice would I give?

Do your homework, practice saying your main points again and again and again, and
try to think of a rebuttal to every counter-argument you can think of.
Martial every last bit of evidence that you have and which may (depending of what rebuttals are made [see above]) and lodge all of these documents with the court by the specified date.
Make sure your evidence/documents are ordered and numbered, and that you can find them easily, even in a pressure situation. Perhaps have a master index (not lodged previously with the court, but with you when you go to court) so you can say to the court, for example, ‘Can we look at document number 47. This is my receipt for the work done, and shows…’
Stay as calm as you can, don’t raise your voice, don’t have a stand-up row with your opponent. Let him/her do he shouting and losing of their temper - judges tend not to be impressed by such behaviour :-)
Lodge all the documents you might ever need with the court. You won’t be penalised for the number of documents you lodge (courts are all used to dealing with vast amounts of paperwork).
Best of luck, and don’t be scared. The system works!

**Comment by Morningstar88** [comment]
Yes. A couple of times, one of which I’ve already mentioned elsewhere so won’t go into again. The other involved a company who tried to stiff me for the final payment for an IT project I did for them.

Background - I was contracted for a network optimisation and upgrade which paid a decent wedge and payment was in 3 stages following key deliverables being on schedule. The final payment being due on the project being signed off as complete and satisfactory. For an IT project it actually ran pretty smoothly and when the final part went live I was feeling pretty please with myself. There were a couple of issues that were sorted in a couple of hours and resulted from changes within the company that I’d not been told about.

They signed everything off including the statement that they accepted the work and had no further issues. Thank you very much I said, and here’s the final invoice… just shy of 5 figures… payable in 14 days. As the other payments hadn’t been an issue, despite being higher, I assumed this would hit my bank 2 weeks later, only it didn’t. Another week went by and still nothing so I rang them. Apparently someone had marked it as a 30 day payment, very sorry etc, can I let it go till then. I did.

At 30 days I still hadn’t been paid so gave it a couple more days before ringing again… again they tried to fob me off with “It’s in the next payment run”, which was when I wanted to know… oh 2 weeks. Not acceptable as that would make it a month overdue. I suggested they make an immediate transfer to my account, within 7 days to avoid legal action. Which was further emphasised by the e-mail I sent to the heads of accounts & IT, and the CEO. A reply was received from the head of IT stating that therer were numerous problems with the system and they wouldn’t pay until they were sorted. I pointed out he’d signed the work off as complete and satisfactory and that I’d had no notification of or been called in to sort any problems. I also asked for a list of these alleged problems… which never materialised.

I started a claim online for the outstanding balance, costs and daily interest, which immediately went to their legal team who filed for an extension to their response. Not unsurprising. Their response was much the same as the head of IT’s.. sort the problems and we’ll pay you.

I countered with a copy of my e-mail asking for a list of alleged problems and that 3 weeks later as I’d still had nothing, I was of the opinion that they didn’t exist. A court date was set and magically a list of supposed problems appeared as part of the evidence. The judge ruled that inadmissable as the date of production was only 2 days previous.

It only took him 10 minutes to award judgement to me in full, with payment to be made within 7 days.

It still took the threat of escalating it to the High Court to get them to pay.

**Comment by Morningstar88** [comment]
It can be very chaotic. If you have a case in small claims court, if there is any real money involved, hiring an attorney is good idea. If you want to represent yourself, don’t just show up and try the case or defend it. Being unprepared is a recipe for disaster. Prepare. If you are the plaintiff, ask yourself what proves your case and how do you get your proof into evidence? What do you ask your witnesses? If you are the defendant, what are the problems with the plaintiff’s case? Do you have witnesses that support your defense? Also, the court will have reviewed your case and if the court determines that both sides have merit, the court may push you to settle. And here is not the place to stand on principles; cases are not won on emotion or hunches, but on evidence and you could lose. So be prepared to engage in serious settlement discussions and to make (or accept) a reasonable settlement offer. (Some courts have lawyers on standby to act as mediators or referees to help with settlement negotiations - they are great resources and have a lot of savvy so use them if they are available).
